scripts/myfans_dl.py

```
@@ -31,15 +31,34 @@ def DL_File(m3u8_url_download, output_file, input_post_id):
         if not os.path.exists(output_folder):
             os.makedirs(output_folder)
 
-        result = subprocess.run(
-            ["ffmpeg", "-n", "-i", m3u8_url_download, "-c:v", "copy", "-c:a", "copy", "-loglevel", "quiet", output_file],
+        # Step 1: Download as .ts file first
+        ts_file = output_file.replace('.mp4', '.ts')
+        
+        # Download with longer timeout since this is the network-heavy part
+        download_result = subprocess.run(
```

Ruff (0.8.2)

Local variable `download_result` is assigned to but never used

Remove assignment to unused variable `download_result`

(F841)